graphs/fit_atg_model.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass
+from typing import List
+
+import numpy as np
+import pytest
+from scipy.optimize import least_squares
+
+# TODO(miskosz): Pass in as a parameter?
+EXPONENT = 6.23
```

Comment:
I think we prefer optional parameter with default value and use argparse

Reply:
This file is going to be imported, not invoked from a command line. It is a constant of the model though, the inelegant part is that now it lives in two files.

A little refactoring (coming soon) should solve the problem :)